contractcourt/breach_arbitrator_test.go

```diff
@@ -1203,7 +1203,7 @@ func TestBreachCreateJusticeTx(t *testing.T) {
 		// Create a fake breached output for each type, ensuring they
 		// have different outpoints for our logic to accept them.
 		op := breachedOutputs[0].outpoint
-		op.Index = uint32(i)
+		op.Index = uint32(1000 + i)
```

Comment:
this seems suspiciously unrelated.

Reply:
Updated the comments.

Reply:
Wait so we add to the output index to prevent it from being equal to `0000000000000000000000000000000000000000000000000000000000000000:0`? If so then why is `breachedOutputs[0]` the empty outpoint in the first place?

Reply:
Right, but why don't we specify it? That's my contention here.